pkg/snowflake/external_table.go

```diff
@@ -113,7 +113,7 @@ func (tb *ExternalTableBuilder) Create() string {
 	q.WriteString(fmt.Sprintf(` (`))
 	columnDefinitions := []string{}
 	for _, columnDefinition := range tb.columns {
-		columnDefinitions = append(columnDefinitions, fmt.Sprintf(`"%v" %v AS %v`, EscapeString(columnDefinition["name"]), EscapeString(columnDefinition["type"]), EscapeString(columnDefinition["as"])))
+		columnDefinitions = append(columnDefinitions, fmt.Sprintf(`"%v" %v AS %v`, EscapeString(columnDefinition["name"]), EscapeString(columnDefinition["type"]), columnDefinition["as"]))
```

Comment:
I wonder if a more sustainable fix is to make it so that you can do `columnDefinition.Get("name")` and it'll return behind the scenes `EscapeString(columnDefinition["name"])`

Reply:
Hmm `columnDefinition` is a map of strings so there's no `Get` method.

Reply:
it is now, but it doesn't have to be. I don't think you should make that change in the PR though

Reply:
Ohhh you were saying to change the structure of `columnDefinitions` so that we could use `Get`...

Reply:
What would the difference be from what I changed it to?